src/policy/marksweepspace/native_ms/block_list.rs

```diff
@@ -167,7 +167,7 @@ pub(crate) const MI_BIN_FULL: usize = MAX_BIN + 1;
 pub(crate) const MAX_BIN: usize = 48;
 
 /// Largest object size allowed with our mimalloc implementation, in bytes
-pub(crate) const MI_LARGE_OBJ_SIZE_MAX: usize = MAX_BIN_SIZE;
+pub(crate) const MI_LARGE_OBJ_SIZE_MAX: usize = Block::BYTES;
```

Comment:
`MAX_BIN_SIZE` is `8194 * MI_INTPTR_SIZE` (64K on 64bits, 32K on 32bits). So the max LO size is 64K on 64bits, and 32K on 32bits. After this change, the max LO size is always 64K. In that case, any object with its size between 32K and 64K is allocated with the free list allocator. But the allocator does not have bins for those sizes. That's why we see the out-of-bound error in the CI tests.

Reply:
This could be a simple fix.
```suggestion
pub(crate) const MI_LARGE_OBJ_SIZE_MAX: usize = crate::util::rust_util::min_of_usize(Block::BYTES, MAX_BIN_SIZE);
```